plugins/pixart-rf/fu-pxi-device.c

```diff
@@ -887,7 +887,7 @@ fu_pxi_device_setup_guid (FuPxiDevice *self, GError **error)
 		devid2 = g_strdup_printf ("HIDRAW\\VEN_%04X&DEV_%04X&MODEL_%s",
 					 (guint) hid_raw_info.vendor,
 					 (guint) hid_raw_info.product,
-					 dev_name->str);
+					 model_name->str);
```

Comment:
Why not add both? i.e. create a `devid3` with `model_name`? Then you can choose the most correct GUID in the firmware file.

Reply:
I think it was duplicate with devid in code. That is my fault . I didn't notice this part.
So create devid3 is the better way?

Reply:
If in doubt, more GUIDs are better. If you add the extra one, be sure to update README.md too.

Reply:
After I check model name. it already existed in README. So I think fix code  is better than adding new device id.

Reply:
@hardboprobot what do you think here?